src/main/java/gov/nih/nci/evs/api/model/IncludeParam.java

```diff
@@ -320,15 +320,27 @@ public String[] getExcludedFields() {
     fields.add("stemName"); // stemName always excluded
     if (!synonyms) {
       fields.add("synonyms");
+    } else {
+      fields.add("synonyms.normName");
```

Comment:
I'm still getting a normName in Synonym for the SKD getPropertyTest 
The field `normName` in the JSON string is not defined in the `Synonym` properties. JSON: {"name":"BioCarta ID","normName":"biocarta id","type":"Display_Name","typeCode":"P107"}